server/src/main/java/org/eclipse/openvsx/LocalRegistryService.java

```diff
@@ -172,11 +183,11 @@ public ReviewListJson getReviews(String namespace, String extensionName) {
             throw new NotFoundException();
         var list = new ReviewListJson();
         var serverUrl = UrlUtil.getBaseUrl();
-        list.postUrl = createApiUrl(serverUrl, "api", extension.getNamespace().getName(), extension.getName(), "review");
-        list.deleteUrl = createApiUrl(serverUrl, "api", extension.getNamespace().getName(), extension.getName(), "review", "delete");
-        list.reviews = repositories.findActiveReviews(extension)
-                .map(extReview -> extReview.toReviewJson())
-                .toList();
+        list.postUrl = createApiUrl(serverUrl, "api", extension.getNamespace().getName(), extension.getName(),
+                "review");
+        list.deleteUrl = createApiUrl(serverUrl, "api", extension.getNamespace().getName(), extension.getName(),
+                "review", "delete");
+        list.reviews = repositories.findActiveReviews(extension).map(extReview -> extReview.toReviewJson()).toList();
```

Comment:
Please don't use the autoformatter, it does not help. See also other formatting changes in this file.